hvplot/converter.py

```diff
@@ -2178,6 +2178,12 @@ def _process_tiles_without_geo(self, data, x, y):
             max_x = np.max(data[x])
             min_y = np.min(data[y])
             max_y = np.max(data[y])
+
+            if hasattr(min_x, 'compute'):
+                min_x, max_x = min_x.compute(), max_x.compute()
+            if hasattr(min_y, 'compute'):
+                min_y, max_y = min_y.compute(), max_y.compute()
```

Comment:
This is not a good idea for a number of reasons:

1. Adding additional compute calls can potentially be VERY expensive. Recall when you implemented `symmetric=True` and I suggested that we should not do automatic computations like this because they can potentially run extra computations on terrabytes of data.
2. Even if we decided this was a good idea, you are running four completely separate computes here!!! Dask has ways to fuse multiple calls like this into a single pass over the data.

Reply:
@philippjfr what do you suggest we do instead?

1. Force the user to set the projection in this case? (Ibis data too I guess)
2. Would it be sufficient to check that just a few values (or even 1) are within the bounds?

Reply:
1. Yes, I think we need to document that certain automatic behaviors are not enabled by default on lazy datasets.
2. Maybe but not super convinced, you'd have to write logic to fetch only one chunk I guess.